api/src/test/java/org/pmiops/workbench/cdr/CdrVersionServiceTest.java

```diff
@@ -304,7 +313,7 @@ private void testGetCdrVersionsHasDataType(Predicate<CdrVersion> hasType) {
     assertThat(cdrVersions.stream().anyMatch(hasType)).isFalse();
 
     makeCdrVersion(
-        5L, true, "Test CDR With Data Types", registeredTier, "wgs", true, true, true, true);
+        5L, true, "Test CDR With Data Types", registeredTier, "wgs", true, true, true, true, true);
```

Comment:
so, makeCdrVersion-> sets `tanagraEnabled` to true, but I do not see a check to test a call to check if `getTanagraEnabled` from `DbCdrVersion` returns `true`